src/github/pullRequestModel.ts

```diff
@@ -982,9 +982,9 @@ export class PullRequestModel extends IssueModel<PullRequest> implements IPullRe
 					return {
 						id: context.id,
 						url: context.targetUrl,
-						avatar_url: context.avatarUrl,
+						avatar_url: context.checkSuite.app?.logoUrl,
```

Comment:
Execution will only fall into this `else` if we have determined that the run is not a `CheckRun`, so at this point `context` is `any`. It seems though that we are assuming that really we do want pending runs to be treated as being a `CheckRun`. Would it make sense instead to change this:

https://github.com/burkeholland/vscode-pull-request-github/blob/fbff4e8582138f9272ac0be3f8a81eeb2294cc8a/src/github/graphql.ts#L593-L595

to be:

```ts
export function isCheckRun(x: CheckRun | StatusContext): x is CheckRun {
	return !!(x as CheckRun).checkSuite;
}
```
?